docs/_documentations/installoncloud.md

```diff
@@ -12,94 +12,47 @@ parent: root
 
 # Setting up a Codewind-ready install of Che
 
-To setup a Codewind-ready install of Che, follow these instructions. Alternatively, the install script at https://github.com/eclipse/codewind-che-plugin/tree/master/setup/install_che can be used instead.
+To setup a Codewind-ready install of Che, follow these instructions.
```

Comment:
Sorry I didn't notice this before...but could we say, "Set up a Codewind-ready installation of Che."?

Reply:
@sishida Do we need it at all? It'd be the same as the title

Reply:
@sishida I'm thinking we should also change the title to "Installing Codewind on Eclipse Che"

Reply:
@johnmcollier I like this title. Let's go with "Installing Codewind on Eclipse Che". :)